.github/workflows/scripts/run-integration-tests-group.sh

```diff
@@ -36,11 +42,20 @@ if [[ -z "$TOTAL" ]]; then
     exit 1
 fi
 
+export IMAGE_TAG_RACE=$(make image-tag-race)
+export MIMIR_IMAGE="grafana/mimir:$TAGPREFIX$IMAGE_TAG_RACE"
+export IMAGE_TAG=$(make image-tag)
+export MIMIRTOOL_IMAGE="grafana/mimirtool:$IMAGE_TAG"
+export MIMIR_CHECKOUT_DIR="/go/src/github.com/grafana/mimir"
```

Comment:
We don't necessarily need to modify this file or add `--tagprefix`, we could do this by setting the environment variables as we expect in the integration test. I think given this is temporary it would be better to just set the env vars, but I'm happy either way.

If we do keep it like this though, we should honour the env-vars if they are set. For example (untested):

```suggestion
export IMAGE_TAG_RACE=${IMAGE_TAG_RACE:-$(make image-tag-race)}
export MIMIR_IMAGE=${MIMIR_IMAGE:-"grafana/mimir:$TAGPREFIX$IMAGE_TAG_RACE"}
export IMAGE_TAG=${IMAGE_TAG:-$(make image-tag)}
export MIMIRTOOL_IMAGE=${MIMIRTOOL_IMAGE:-"grafana/mimirtool:$IMAGE_TAG"}
export MIMIR_CHECKOUT_DIR=${MIMIR_CHECKOUT_DIR:-"/go/src/github.com/grafana/mimir"}
```